services/sketch/README.md

```diff
@@ -0,0 +1,8 @@
+# Sketch
+
+[![Sketch's Logo][producti]][product]
+
+Current Sparkbox design tool of choice.
```

Comment:
Is this still true? Again, might be good to have a designer look at this to confirm. With all the raving about Figma it's hard to tell anymore 😄